src/twisted/spread/test/test_banana.py

```diff
@@ -9,7 +9,7 @@
 from twisted.trial.unittest import TestCase
 from twisted.spread import banana
 from twisted.python import failure
-from twisted.python.compat import long, iterbytes, _bytesChr as chr
+from twisted.python.compat import long, iterbytes
```

Comment:
`long` can go, and be replaced by `int`.  You can do that in this PR, or a separate one if you want.